library/src/main/java/com/loopj/android/http/AsyncHttpClient.java

```diff
@@ -251,6 +251,11 @@ public void process(HttpResponse response, HttpContext context) {
         httpClient.addRequestInterceptor(new HttpRequestInterceptor() {
             @Override
             public void process(final HttpRequest request, final HttpContext context) throws HttpException, IOException {
+
+                AuthSchemeRegistry authSchemeRegistry = new AuthSchemeRegistry();
+                authSchemeRegistry.register("Bearer", new BearerAuthSchemeFactory());
+                context.setAttribute(ClientContext.AUTHSCHEME_REGISTRY, authSchemeRegistry);
```

Comment:
I don't like this line, it means we're overriding any AUTHSCHEME_REGISTRY already contained by underlying HttpContext ? This should be enabled on-demand, not as only AuthScheme available from library